tests/Repository.py

```diff
@@ -693,3 +693,47 @@ def testOwner(self):
         owner = lazy_repo.owner
         eager_repo = self.getEagerRepository()
         self.assertEqual(owner, eager_repo.owner)
+
+    def testEnableVulnerabilityAlert(self):
+        lazy_repo = self.getLazyRepository()
+        self.assertEqual(lazy_repo.enable_vulnerability_alert(), True)
```

Comment:
All of these can use either self.assertTrue() and self.assertFalse() to simply the assertions.

Reply:
Updated in the new commit.